airgun/views/container.py

```diff
@@ -0,0 +1,150 @@
+from widgetastic.widget import (
+    Text,
+    TextInput,
+    View,
+    Checkbox,
+)
+from airgun.views.common import (
+    BaseLoggedInView,
+    SearchableViewMixin,
+    SatTab,
+)
+from airgun.widgets import (
+    FilteredDropdown,
+    ActionsDropdown,
+    SatTable,
+    MultiSelect,
+    EditableEntryCheckbox,
+)
+from widgetastic_patternfly import BreadCrumb
+
+
+class ContainerView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Containers']")
+    new = Text("//a[contains(@href, '/containers/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Name': Text('./a'),
+            'Status': Text("./span[text()='On']"),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class ContainerCreateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Steps'
+            and self.breadcrumb.read() == 'Create container'
+        )
+
+    @View.nested
+    class ContainerPreliminaryCreateView(BaseLoggedInView):
+        next_step = Text("//button[contains(@id, 'next')]")
+
+        @View.nested
+        class compute_resource(SatTab):
+            TAB_NAME = 'Compute resource'
+            deploy_on = FilteredDropdown(
+                id='s2id_docker_container_wizard_states_preliminary_compute_resource_id')
+
+        @View.nested
+        class locations(SatTab):
+            TAB_NAME = 'Locations'
```

Comment:
redundant line; when tab name is equal to class name capitalized - there's no need to specify `TAB_NAME` explicitly